lms/templates/ccx/schedule.underscore

```diff
@@ -66,32 +71,39 @@
                 <span class="sr"><%- gettext('Click to change') %></span>
               </button>
             </td>
-            <td><button class="remove-unit ccx-button-link" aria-label="Remove subsection <%= child.display_name %>">
+            <td><button class="remove-unit ccx-button-link" aria-label="<%- interpolate(
+                gettext('Remove subsection %(subsectionDisplayName)s'), {subsectionDisplayName: child.display_name}, true) %>">
               <i class="fa fa-remove" aria-hidden="true"></i> <%- gettext('remove') %>
             </button></td>
           </tr>
           <% _.each(child.children, function(subchild) { %>
-            <tr class="vertical" data-dapth="3"
+            <tr class="vertical" data-depth="3"
                 data-location="<%= chapter.location %> <%= child.location %> <%= subchild.location %>">
               <td class="unit">&nbsp;
                  <span class="sr"><%- gettext('Unit') %>&nbsp;</span>
                  <%= subchild.display_name %>
               </td>
-              <td class="date start-date">
-                <button class="ccx-button-link">
+              <td class="date start-date no-link">
+                <% if (subchild.start) { %>
                   <%= subchild.start %>
-                  <span class="sr"><%- gettext('Click to change') %></span>
-                </button>
+                <% } else { %>
+                  <%- gettext('Unscheduled') %>
+                <% } %>
               </td>
-              <td class="date due-date">
-                <button class="ccx-button-link">
+              <td class="date due-date no-link">
+                <% if (subchild.due) { %>
                   <%= subchild.due %>
-                  <span class="sr"><%- gettext('Click to change') %></span>
+                <% } else { %>
+                  <%- gettext('Unscheduled') %>
```

Comment:
Add a translators comment here - what does "unscheduled" mean? Generally when there's just a single word or short phrase, you should add some context with a translator's comment, because translators won't have the context of the surrounding code and may provide an incorrect translation inadvertantly.